.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags: [v*]
```

Comment:
Can you make this `[tst-v*]` so a) we can play around with it and test it and b) it won't interfere with the current release process?

Reply:
done